Italian/caramelle.md

```diff
@@ -1 +1,6 @@
-Una mattina ti svegli in una stanza con le pareti ricoperte di caramelle.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.